.travis.yml

```diff
@@ -1,40 +0,0 @@
-language: php
-matrix:
-  fast_finish: true
-  include:
-    - php: 7.3
-      env: DB=mysql BEHAT=true
-    - php: 7.4
-      env: DB=mysql BEHAT=true
-cache:
-  directories:
-    - $HOME/.composer/cache
-before_install:
-  - export FLOW_TARGET_VERSION=master
-  - if [ "$DB" = "pgsql" ]; then sudo mount -o remount,size=25% /var/ramfs ; fi
-  - phpenv config-rm xdebug.ini
-  - env
-  - sudo add-apt-repository ppa:git-core/ppa -y
-  - sudo apt-get update
-  - sudo apt-get install git -y
-  - cd ..
-  - git clone https://github.com/neos/flow-development-distribution.git -b ${FLOW_TARGET_VERSION}
-  - cd flow-development-distribution
-  - composer update
-install:
-  - rm -rf Packages/Application/Neos.Behat
-  - mkdir -p Packages/Application/Neos.Behat
-  - cp -R $TRAVIS_BUILD_DIR/* Packages/Application/Neos.Behat
-  - composer dump-autoload
-before_script:
-  - echo 'date.timezone = "Africa/Tunis"' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
-  - echo 'opcache.fast_shutdown = 0' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
-  - echo 'opcache.enable_cli = 0' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
-  - echo 'zend.enable_gc = 0' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
-  - echo 'report_zend_debug = 0' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
-  - echo 'report_memleaks = 0' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
-  - rm -rf Build/Behat
-  - php flow behat:setup
-  - php flow behat:kickstart --packageName "Neos.Behat" --host "http://localhost/"
-script:
-  - bin/behat --stop-on-failure -f progress -c Packages/Application/Neos.Behat/Tests/Behavior/behat.yml
```

Comment:
We don't use travis any more, but this is now removed without replacement. Maybe a simple `.github` action could be added again?

Reply:
I think we'll notice if it breaks – as the Neos and Flow tests should then fail, no?

Reply:
yes im not fluid in gh actions and like to not do it ^^

Reply:
> I think we'll notice if it breaks

yes, that's true :)